Modules/IO/Meta/test/itkLargeMetaImageWriteReadTest.cxx

```diff
@@ -73,17 +71,13 @@ ActualTest(std::string filename, typename TImageType::SizeType size)
 
     std::cout << "Initializing pixel values " << std::endl;
 
-    IteratorType itr(image, region);
-    itr.GoToBegin();
-
-    pixelValue = itk::NumericTraits<PixelType>::ZeroValue();
-
+    PixelType pixelValue = itk::NumericTraits<PixelType>::ZeroValue();
```

Comment:
Should this be `PixelType pixelValue{};`?

Reply:
Sure. but somewhat unrelated.